app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -1,86 +0,0 @@
-<div class="form_row">
```

Comment:
Did you removed this on purpose?

Reply:
Well yes.
I noticed the file was moved to templates/partials/calendar/event so I thought this was left over accidentally as it wasn't being loaded from anywhere.